definitions/declarations/httparchive.js

```diff
@@ -1,17 +1,41 @@
-const stagingTables = ['pages', 'requests', 'parsed_css']
-for (const table of stagingTables) {
+// Staging tables source: https://github.com/HTTPArchive/crawl/blob/main/crawl.py
+['pages', 'requests', 'parsed_css'].forEach(table =>
   declare({
     schema: 'crawl_staging',
     name: table
   })
-}
+)
 
-declare({
-  schema: 'wappalyzer',
-  name: 'technologies'
-})
+assert('corrupted_technology_values')
```

Comment:
Can we add an link to a GitHub issue where we explain why this is necessary, and aim to close it out eventually once the root cause is found?

Reply:
```suggestion
// See https://github.com/HTTPArchive/dataform/issues/43
assert('corrupted_technology_values')
```